needls/server.py

```diff
@@ -510,6 +512,23 @@ def update_settings(ls, *args):
         )
         return
 
+    if ext_conf_py_file:
+        # Check if given extra configuration file name and location is different with conf.py
+        if ext_conf_py_file == os.path.join(docs_root, "conf.py"):
```

Comment:
Is this a problem?
I mean there should be use only one, right?
So if ext_conf_py is given, the normal `conf.py` should not be used anywhere.

Or see above. It could be allowed to have `ext_conf.py_file` be set to conf.py.
Just rename the var to `conf_file` or something similar :)